openwisp_monitoring/tests/test_selenium.py

```diff
@@ -89,6 +89,8 @@ def test_restoring_deleted_device(self):
         org = self._get_org()
         self._create_credentials(auto_add=True, organization=org)
         device = self._create_config(organization=org).device
+        # Delete iperf check to prevent unnecessary response timeout
+        Check.objects.filter(check_type__endswith='Iperf').delete()
```

Comment:
@Aryamanz29 can you share why this is needed?

Reply:
While running tests, I'm getting `TimeoutException` due to extra check, Now it's working fine, I'll remove that line :+1: 
```py
def test_restoring_deleted_device(self):
...
...       
      except TimeoutException:
            self.fail('Failed saving device')
```